.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+  schedule:
+    - cron:  '0 5 * * FRI'
```

Comment:
```suggestion
  schedule:
    - cron:  '0 5 * * FRI'
```
Any reason for doing a scheduled run?

Reply:
Nothing in particular, just so get a test run at least once a week in case the containers changed.

Reply:
Removed.